catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/LibraryImpl.java

```diff
@@ -254,7 +254,7 @@ public LibraryManager get_manager(String manager_type, AccessCriteria access_cri
                     poa.activate_object_with_id(managerId.getBytes(Charset.forName(NsiliEndpoint.ENCODING)),
                             standingQueryMgr);
                 } catch (ServantAlreadyActive | ObjectAlreadyActive | WrongPolicy e) {
-                    LOGGER.error("Error activating StandingQueryMgr: {}", e);
+                    LOGGER.info("Error activating StandingQueryMgr: ", e);
```

Comment:
The reason why they are different is because the LibraryImpl is the one creating the CreationMgrImpl and I didn't want to display multiple stack traces all at info. So you should get just the one message.